features/main_landing_page.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Main landing page
+  As a visitor
+  In order to get an overview over the service
+  I need to see a main landing page
+
+  Scenario: Visitor visits main page
+    When I visit the main landing page
+    Then I should see "Welcome to HomeChef"
+    And I should see input of type "text" with name "enter_location"
+    And I should see a "general_random_dishes" placeholder
+    And I should see a "logotype" placeholder
+    And I should see a link called "Home"
```

Comment:
Yes, we had the link and the click of it in one statement before, but were required to split them. Should we omit the naming of the link we are clicking altogether? How can we make sure it is THAT specific one link that leads us to that specific page?